Tutorials (#221)
applied-material-modeling/neml2: doc/tutorials/extension/model_visualization/main.md

@insert-title:tutorials-extension-model-visualization

[TOC]

## Motivation

> During the process of model development, it is of general interest (and is also very much encouraged) to reuse existing models. This is particularly useful and effective owing to the extreme modular design of NEML2, i.e., model composition.

Reviewer: ```suggestion
Users should reuse existing NEML2 models as much as possible. This is particularly effective owing to the extremely modular design of NEML2, i.e., model composition.
```
